src/main/java/reborncore/api/power/ExternalPowerManager.java

```diff
@@ -30,18 +30,21 @@
 
 import net.minecraft.item.ItemStack;
 import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.EnumFacing;
 import reborncore.common.powerSystem.TilePowerAcceptor;
+import reborncore.common.powerSystem.forge.ForgePowerItemManager;
 
 public interface ExternalPowerManager {
 
 	public ExternalPowerHandler createPowerHandler(TilePowerAcceptor acceptor);
 
 	public boolean isPoweredItem(ItemStack stack);
 
-	public boolean isPoweredTile(TileEntity tileEntity);
+	public boolean isPoweredTile(TileEntity tileEntity, EnumFacing side);
 
 	public void dischargeItem(TilePowerAcceptor tilePowerAcceptor, ItemStack stack);
+	public void dischargeItem(ForgePowerItemManager powerAcceptor, ItemStack stack);
```

Comment:
Just trying to get my head around this. I cannot see where its being called?

Is this for charging TR items when the FE support is disabled?

Reply:
Yeah, currently this function would not actually be called from anywhere. Its purpose would be a TR item pullling charge from a non TR item, which currently does not happen anywhere. However I added it because I thought that it would be inconsistent to not have it when the chargeItem variant does exist. While chargeItem will be used for lithium packs, this still wont be used, since I intend to add an
(empty by default) requestEnergyFromArmor method that would allow TR items to pull energy from IC2 battery packs, instead of having TR items directly discharge them.

If you’d like, I can replace the dischargeItem function with requestEnergyFromArmor at this moment, to minimize breakage.